test/micro-distribution/lib/Standard/Base/0.0.0-dev/src/Runtime.enso

```diff
@@ -52,3 +52,6 @@ with_disabled_context_builtin : Context -> Text -> Function -> Any
 with_disabled_context_builtin context environment ~action = @Builtin_Method "Runtime.with_disabled_context_builtin"
 
 primitive_get_stack_trace = @Builtin_Method "Runtime.primitive_get_stack_trace"
+
+value_for_uuid id = instrumentor_builtin "uuid" id
+instrumentor_builtin op args = @Builtin_Method "Meta.instrumentor_builtin"
```

Comment:
Do we not need these in the full distribution too?

Reply:
The whole concept of _micro distribution_ is flawed 
- it is just a random selection of stuff from `Standard.Base` to make sure some unit tests work
- ideally we have #8852 
- the _prelude_ would be a subset of `Standard.Base`
- the micro distribution is deleted and _prelude_ is used instead

Until that happens, we just have to update micro-distribution to be on par with builtins used by `Standard.Base`.